optuna/distributions.py

```diff
@@ -91,6 +91,9 @@ class UniformDistribution(
         BaseDistribution):
     """A uniform distribution in the linear domain.
 
+    This object is instantiated by :func:`~optuna.trial.Trial.suggest_uniform`, and passed to
```

Comment:
In my opinion, it's better to add "typically" (or another similar word) to this sentence, because methods other than `suggest_xxx` (e.g., `product_search_space`) may also create distributions objects.

Reply:
Thanks! That makes sense.